src/rest/entities.ts

```diff
@@ -134,7 +134,11 @@ async function resolveRichTextLinks(
   entityReferenceMap: EntityReferenceMap,
   locale: string
 ) {
-  if (node.nodeType.includes('embedded')) {
+  if (
+    node.nodeType.includes('embedded') ||
+    node.nodeType === 'entry-hyperlink' ||
```

Comment:
Will investigate if we have to add this in GraphQL as well, will create a followup PR if necessary

Reply:
What about asset and resource links? Also, at this point maybe it's better to use `@contentful/rich-text-types` to get node type ids

Reply:
assets are supported (`node.nodeType.includes('embedded')` includes the embedded-asset-block + we have the asset-hyperlink)
I'll refactor it to use the types from the package. Not sure about resource links, since they will open in a new tab, so live updates will not work there?

Reply:
> assets are supported (node.nodeType.includes('embedded') includes the embedded-asset-block + we have the asset-hyperlink)

I'm sorry. I was probably too busy to read the PR carefully. You're right.

Reply:
> Not sure about resource links, since they will open in a new tab, so live updates will not work there

If resources are not relevant, do we need to maybe exclude `embedded-resource-block` from being included with `includes('embedded')`?

Reply:
I am not sure, do you mind if we create a follow up ticket for that? We need to investigate either way how embedded resources links work in live preview, (we haven't tested it out yet)

Reply:
https://contentful.atlassian.net/browse/TOL-1293